config/isolatedStarAssociation.py

```diff
@@ -1,2 +1,2 @@
 # irgzyN921 is preferred order of Eli and tested in RC2
-config.band_order = ["i", "r", "g", "z", "y", "N921", "N816", "N1010", "N387", "N515"]
+config.band_order = ["i", "r", "g", "z", "y", "N921", "N816", "N1010", "N387", "N515", "N395"]
```

Comment:
I doubt it matters much but in practice is there some logic to the order of the narrow bands here?